java/src/main/c++/vowpalWabbit_learner_VWLearners.cc

```diff
@@ -26,6 +26,20 @@ JNIEXPORT void JNICALL Java_vowpalWabbit_learner_VWLearners_closeInstance(JNIEnv
   }
 }
 
+JNIEXPORT void JNICALL Java_vowpalWabbit_learner_VWLearners_saveModel(JNIEnv *env, jclass obj, jlong vwPtr, jstring filename)
+{ try
+  {
+    const char* utf_string = env->GetStringUTFChars(filename, NULL);
```

Comment:
This looks better.  The way it's written now the JNI lines (everything other than `save_predictor`) doesn't even need to be in the `try` block.